subsys/bluetooth/host/audio/chan.c

```diff
@@ -2408,78 +2429,65 @@ int bt_audio_broadcast_sink_sync(struct bt_audio_broadcast_sink *sink,
 		return -EAGAIN;
 	}
 
+	CHECKIF(sink->big_encrypted && broadcast_code == NULL) {
+		BT_DBG("Broadcast code required");
+		return -EINVAL;
+	}
+
 	/* Validate that number of bits set is less than number of channels */
 	/* 0x1f is maximum number of BIS indexes */
```

Comment:
I can not see the 0x1f being used any more, so this line of the comment can be removed.